app/bundles/LeadBundle/EventListener/CampaignActionAnonymizeUserDataSubscriber.php

```diff
@@ -0,0 +1,506 @@
+<?php
+
+namespace Mautic\LeadBundle\EventListener;
```

Comment:
```suggestion

declare(strict_types=1);

namespace Mautic\LeadBundle\EventListener;
```
Missed strict type checking.